.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
remove?

Reply:
Mistaken IDE files. Removed.

Reply:
feel free to add these types of files to moveit's .gitignore